src/Swarm/TUI/Controller.hs

```diff
@@ -320,6 +320,8 @@ handleMainEvent ev = do
     -- toggle creative mode if in "cheat mode"
     ControlChar 'v'
       | s ^. uiState . uiCheatMode -> gameState . creativeMode %= not
+    -- toggle collapse/expand REPL
+    ControlChar 'k' -> uiState . uiShowREPL %= not
```

Comment:
As I mentioned in a comment elsewhere Ctrl-k is actually also taken (though it's not obvious from the Swarm source code because it's one of the default keybindings provided by the brick editor widget: https://hackage.haskell.org/package/brick-1.6/docs/Brick-Widgets-Edit.html ).

Reply:
Ah, thank you.  I have changed it to Ctrl-s.  :crossed_fingers: